src/store/modules/TsModsModule.ts

```diff
@@ -45,6 +45,7 @@ function isPackageListChunk(value: unknown): value is PackageListChunk {
 }
 
 const EXCLUSIONS = 'https://raw.githubusercontent.com/ebkr/r2modmanPlus/master/modExclusions.md';
+const PARTIAL_UPDATE_ERROR = 'Failed to fetch and cache some of the packages.';
```

Comment:
I think this message could be made more informative for the users. Instead of "fetch and cache" we could use the "refresh online mod list" used elsewhere in the UI, and we could explain how this affects the users. Something like:
"Failed to fully refresh the online mod list. Some mod versions might be unavailable."

It's not perfect since it doesn't tell the user whether it was one package out of one, two, or twenty that failed. But it'll probably do for now.